src/chorizo-sweet-potato-enchiladas.md

```diff
@@ -0,0 +1,37 @@
+# Chorizo and Sweet Potato Enchiladas
+
+Very filling with a sweet and spicy flavour.
+
+## Ingredients
+
+- 1 large sweet potato (approx 1 pound)
+- 1 bell pepper
+- 1 jalapenos (adjust based on spice preference)
+- 2 cloves garlic
+- vegetable oil
+- 3/4 pounds chorizo (mexican chorizo is preferred)
+- 8 flour tortillas
+- 1.5 cups grated cheddar or other cheese
```

Comment:
```suggestion
- 1.5 cups ( 120g ) grated cheddar or other cheese
```

Reply:
It depends on the chorizo you use and the consistency of it, if you are using cured chorizo then you will need to chop it, otherwise you should just be able to crumble it in.